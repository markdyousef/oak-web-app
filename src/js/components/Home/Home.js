import React, { Component } from 'react';

import logo from 'img/clai-logo.png';

import css from './Home.css';


class Home extends Component {
    static propTypes = {}

    constructor() {
        super();
        this.state = {};
    }

    render() {
        return (
            <div className={css.container}>
                <h1>Home</h1>
                <img src={logo} role="presentation" />
            </div>
        );
    }
}

export default Home;
