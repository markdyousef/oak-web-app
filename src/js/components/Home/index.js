const React = require('react');

/* class HomePage extends React.Component {
    render() {
        return (
            <div>Home</div>
        );
    }
} */


function HomePage() {
    return (
        <div>Home</div>
    );
}

export default HomePage;
