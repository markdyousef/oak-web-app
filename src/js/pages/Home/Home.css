:root {
    --primary-color: #5ed892;
    --main-black-color: #131517;
    --light-black-color: #2d3438;
    --dark-gray-color: #737a80;
    --light-gray-color: #f2f2f2;
}

.container {
    width: 100%;
    display: flex;
    flex-direction: column;
}

.stats {
    width: 100%;
    height: 200px;
    background-color: #fff;
    box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16), 0 3px 6px rgba(0, 0, 0, 0.23);
}
