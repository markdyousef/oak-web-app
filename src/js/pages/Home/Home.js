// @flow
import React, { Component, PropTypes } from 'react';
import Collections from '../../components/Collections';
import TopNav from '../../components/TopNav';

import css from './Home.css';

class Home extends Component {
    static propTypes = {};
    static defaultProps = {};
    constructor() {
        super();
        this.state = {};
    }
    render() {
        return (
            <div className={css.container}>
                <TopNav />
                <div className={css.categories}>
                    <Collections />
                </div>
            </div>
        );
    }
}

export default Home;
