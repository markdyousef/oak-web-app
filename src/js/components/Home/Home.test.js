import React from 'react';
// import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Home from './Home';


// describe('Home', () => {
//     const wrapper = shallow(<Home />);
//
//     it('renders as a <div>', () => {
//         expect(wrapper.type()).to.eql('div');
//     });
//
//     it('contains a header', () => {
//         expect(wrapper.find('h1')).to.have.length(1);
//     });
// });

test('Home renders', () => {
    const component = renderer.create(
        <Home />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
