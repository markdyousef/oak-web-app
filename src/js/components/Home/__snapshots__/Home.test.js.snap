exports[`test Home renders 1`] = `
<div
  className={undefined}>
  <h1>
    Home
  </h1>
  <img
    role="presentation"
    src="test-file-stub" />
</div>
`;
